commands/instances.go

```diff
@@ -842,29 +842,30 @@ func Upgrade(ctx context.Context, req *rpc.UpgradeRequest, downloadCB DownloadPr
 
 // LoadSketch collects and returns all files composing a sketch
 func LoadSketch(ctx context.Context, req *rpc.LoadSketchRequest) (*rpc.LoadSketchResponse, error) {
-	sketch, err := builder.SketchLoad(req.SketchPath, "")
+	// TODO: This a ToRpc function for the Sketch struct
```

Comment:
I didn't want to introduce breaking changes with this refactoring but I think changing the gRPC interface `LoadSketchResponse` message to reflect the internal `Sketch` struct would be a nice improvement for the future.

Also I think we should start adding `ToRpc/FromRpc` functions to structs that need to be converted when necessary, as of now that are some functions that handle those conversions scattered around different packages.